src/luro.rs

```diff
@@ -12,33 +13,31 @@ use twilight_http::Client;
 use twilight_lavalink::Lavalink;
 use twilight_standby::Standby;
 
+// Auto gen toml folder libs
+use std::fs;
+
 impl Luro {
     /// Initialise and return an instance of Luro
     pub async fn init() -> Result<(Arc<Self>, Vec<Shard>), Error> {
+
+        // Loads dotenv. This allows std::env to view the variables in the file
+        dotenv().ok();
+
         // Luro's Discord token, grabbed from the "DISCORD_TOKEN" environment variabled
-        let token = match env::var("DISCORD_TOKEN") {
-            Ok(ok) => ok,
-            Err(err) => panic!("No DISCORD_TOKEN defined: {err}"),
-        };
+        let token = env::var("DISCORD_TOKEN").expect("No DISCORD_TOKEN defined");
 
         // Lavalink host, defined by the "LAVALINK_HOST" environmental
-        let lavalink_host = match env::var("LAVALINK_HOST") {
-            Ok(ok) => ok,
-            Err(err) => panic!("No LAVALINK_HOST defined: {err}"),
-        };
+        let lavalink_host = env::var("LAVALINK_HOST").expect("No LAVALINK_HOST defined");
+
+        // Lavalink authorisation, defined by the "LAVALINK_AUTHORIZATION" environmental
+        let lavalink_auth = env::var("LAVALINK_AUTHORIZATION").expect("No LAVALINK_AUTHORIZATION defined: {err}");
```

Comment:
Update this to use the british english spelling of 'Authorisation'. Mistake on my part!
```suggestion
        let lavalink_auth = env::var("LAVALINK_AUTHORISATION").expect("No LAVALINK_AUTHORISATION defined: {err}");
```